### PR TITLE
Cookie notice module

### PR DESCRIPTION
**Pull request by threesquared**

### CODE REVIEW

File: src/assets/js/modules/cookie-notice.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+(() => {
```
Comment (mikefrancis) [inline src/assets/js/modules/cookie-notice.js]: I am just wondering if we can utilise the hide/show stuff for this - also should the text (and HTML) not be in the website? What if it needs translating?

File: src/assets/scss/5-objects/_objects.row.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
+// =============================================================================
+//  OBJECTS - ROW
+// =============================================================================
+
+.row {
```
Comment (mikefrancis) [inline src/assets/scss/5-objects/_objects.row.scss]: Is this new grid stuff? What's going on with the old stuff?

File: src/assets/js/modules/cookie-notice.js (hunk 1 above)
Comment (mikefrancis) [reply src/assets/js/modules/cookie-notice.js]: I see - I am just concerned about putting something this specific in the toolkit, but have approved anyway as it seems like we need to ship soon. Maybe we can refactor later, will wait to see what @matine says.

File: src/assets/scss/5-objects/_objects.row.scss (hunk 2 above)
Comment (matine) [reply src/assets/scss/5-objects/_objects.row.scss]: As discussed, have created specific cookie notice styling in a separate component for now. When we have more time we can extend the grid system

**Pull request merged**